Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -60,8 +60,13 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
                 return new MatchResult(false, UserSettingSearchPrecision);
 
             query = query.Trim();
-            TranslationMapping translationMapping;
-            (stringToCompare, translationMapping) = _alphabet?.Translate(stringToCompare) ?? (stringToCompare, null);
+            TranslationMapping translationMapping = null;
+            if (_alphabet is not null && !_alphabet.CanBeTranslated(query))
```

Comment:
why it is `!_alphabet.CanBeTranslated(query)`? If query can be translated then we do not translate the stringtocompare?

Reply:
If a query contains Chinese character, it means user is typing with Chinese IME or pasting Chinese. so definitely no Pinyin in it.

Reply:
okok make sense